add doc on updating bindata in release guide (#4638)
operator-framework/operator-sdk: website/content/en/docs/contribution-guidelines/releasing.md

GitHub releases live under the [`Releases` tab][release-page] in the operator-sdk repo.

### Updating OLM bindata

> At the time of an Operator SDK release, if you would like to add support for a new OLM version, create a PR which updates the `olm bindata` to contain the manifests of the version which needs to be supported. To do so:

Reviewer: ```suggestion
Prior to an Operator SDK release, add bindata for a new OLM version by following these steps:
```

> 1. Update [`OLM_VERSIONS`][olm_version] variable in Makefile, to include the version that needs to be added to the list of supported versions. We generally support latest 3 versions of OLM present at the time of a given Operator SDK release.

Reviewer: ```suggestion
1. Add the new version to the [`OLM_VERSIONS`][olm_version] variable in the Makefile. 
1. Remove the *lowest* version from that variable, as `operator-sdk` only supports 3 versions at a time.
```

2. Run `make bindata`.

> 3. Make sure that the list of supported OLM versions stated in the [`Overview`][overview] section of SDK documentation is updated.

Reviewer: ```suggestion
3. Update the list of supported OLM versions stated in the [`Overview`][overview] docs section.
```
